in IE8 e.which is always 0 better to catch all mouse clicks then none

avipinto opened the pull request:
backbone.intercept didn't intercept any links on IE8 since jQuery sets e.which to 0 for left click on IE8
didn't create an issue for this but i can if you want me to

avipinto committed:
in IE8 e.which is always 0 better to catch all mouse clicks then none

avipinto committed:
added an option to include the queryString along with the pathname - by default it is set to false to maintain  backward compatibility

jmeas commented:
Thanks for the PR, @avipinto!

When you get a chance, can you split this into 2 PRs? And don't worry about doing the build – I'll do that when it's time for a release.

avipinto commented:
removed the version and squashed the commits for the queryString feature but i didn't find a way to split the PR.
when i pushed to master on my fork, i didn't know it will update the PR..
will be happy to split the PR if you have a link to an explanation how to (the only way i can think of is removing this PR, moving each change to a separate branch and then doing a PR, this is how it is done?)
